### PR TITLE
Update theming docs

### PR DESCRIPTION
**Pull request by Juliet-Brown**

### CODE REVIEW

File: packages/govtnz-ds-website/src/docs/basics/themes__examples.md [hunk 1]
```diff
@@ -123,3 +132,9 @@
     </Example>
 
 </ExampleContainer>
+
+## Customising a theme
+
+You can customise the Design System theme to follow your brand guidelines by altering the values in the [Sass variables file](https://github.com/GOVTNZ/govtnz-design-system/blob/master/packages/govtnz-ds/build/scss/_settings.scss).
+
+We encourage you to make the look, feel, and usability of elements as consistent as possible with the Design System. And when you customise, make sure your visual elements continue to meet the NZ [Government Web Accessibility Standard](https://www.digital.govt.nz/standards-and-guidance/nz-government-web-standards/web-accessibility-standard-1-1/).
```
Comment (Juliet-Brown) [inline packages/govtnz-ds-website/src/docs/basics/themes__examples.md:140]: @holloway checking this is the correct place  for theme content. I ran` yarn compile-markdown ` and it didn't seem to update. I will investigate more :)

Comment (holloway) [reply packages/govtnz-ds-website/src/docs/basics/themes__examples.md:140]: @Juliet-Brown Nah it's in `packages/govtnz-ds-website/src/pages/basics/themes.tsx` :)

**Comment by Juliet-Brown** [comment]
@holloway  is it okay that the contact form isn't in react hooks? I assumed that  since most of the project wasn't using it  for me not to use it to keep consistent. But I seen on themes when updating this content you started to use hooks :D .

**Comment by holloway** [comment]
@Juliet-Brown I prefer Hooks, but the Class-based way is good too and probably no point changing it now :)

File: packages/govtnz-ds-website/src/docs/basics/themes__examples.md [hunk 2]
```diff
@@ -31,9 +29,9 @@
                 <FlexRow>
                     <FlexColumn xs="12" sm="12" md="12" lg="12">
                         <H1>Contact us</H1>
-                        <P>Send a message to let us know about the issue you’re having.</P>
-                        <InputBlock type="text" label="What’s your name?" name="anyName" id="anyNameId" width={20}  />
-                        <InputBlock type="email" label="What’s your email address?" name="email" id="anyEmailId" width={20} />
```
Comment (holloway) [inline packages/govtnz-ds-website/src/docs/basics/themes__examples.md:36]: `width={20}` is still needed on both

File: packages/govtnz-ds-website/src/pages/basics/themes.tsx [hunk 3]
```diff
@@ -113,35 +113,30 @@ const ThemesPage = (pageProps: PageRendererProps) => {
             <H1 styleSize="xlarge" id="main-heading">
               Themes
             </H1>
-
-            <P styleSize="large">
+            <p className="g-body g-body-l">
```
Comment (holloway) [inline packages/govtnz-ds-website/src/pages/basics/themes.tsx:116]: We should be using our React components for Paragraphs `P` not `p`

File: packages/govtnz-ds-website/src/pages/basics/themes.tsx [hunk 4]
```diff
@@ -113,35 +113,30 @@ const ThemesPage = (pageProps: PageRendererProps) => {
...
-              Select one of the following 3 theme options to see how Design
-              System components adapt to different colour schemes. Anything you
-              type in the text fields will remain when you switch between
-              themes. No information is sent if you select the ‘Submit’ button.
-            </p>
-
+            </p>{' '}
+            <H2 styleSize="large">Example themes</H2>{' '}
+            <p className="g-body g-body-m">
+              themes Select one of the following 3 theme options to see how
```
Comment (holloway) [inline packages/govtnz-ds-website/src/pages/basics/themes.tsx:124]: "themes Select" ?

**Pull request closed without merging**